Add transliteration menu

akprasad opened the pull request:
A flexible menu that transliterates between two arbitrary scripts.
To keep the tool user-friendly for now, I pared down the list to
remove schemes and encodings that seem less relevant.

Test plan: unit tests and ran the tool on dev.

<img width="260" alt="image" src="https://user-images.githubusercontent.com/1429776/186715059-21bcf914-dc5a-475f-a8e5-9dd402757bb4.png">

akprasad commented:
Assigning @epicfaace as a reviewer since this seems like a tool he would consider using.

akprasad commented:
Also adding @kvchitrapu because this just came up on our Discord.

kvchitrapu reviewed (approved):
LGTM.

kvchitrapu commented:
> When I highlight text and then click on the transliterate button, the highlight disappears. The text still gets transliterated, but the disappearing highlighting is confusing.
@akprasad  @epicfaace Is there a deployment(like vercel) available to testdrive this commit?

kvchitrapu commented:
> 

Never mind, I got it now. It is easy to test on my local setup.

akprasad commented:
> For me, having an integration similar to that (having it easier to type in Devanagari) would be something I would use.

I see. This is certainly more usable for a general audience but would be tough without much more eng work.

> I'm not sure how often something like an IAST -> Devanagari transliteration tool

Ah, that's just an artifact of a lazy copy-paste. I've updated the from and to scripts for more clarity.

> When I highlight text and then click on the transliterate button, the highlight disappears. The text still gets transliterated, but the disappearing highlighting is confusing.

Fixed!

> Is there a deployment(like vercel) available to testdrive this commit?

No, not yet -- for now, you'll have to build locally. Glad you got it working on your end.
